pkg/commands/options/filestuff.go

```diff
@@ -142,3 +155,10 @@ func EnumerateFiles(fo *FilenameOptions) chan string {
 	}()
 	return files
 }
+
+func ResolveSymlink(path string, fi os.FileInfo) (symlink string, err error) {
```

Comment:
Can this be replaced with https://pkg.go.dev/path/filepath#EvalSymlinks ?

Reply:
it didn't work when I switched to EvalSymlink, I don't know why 🤷🏻‍♂️

Reply:
I'd like to understand better why that's the case. This is getting fairly complicated, and I'd rather depend on stdlib methods as much as possible. If we still need to write significant code to handle symlinks, I'm a bit more inclined not to support symlinks for now.